src/user/entities/user.entity.ts

```diff
@@ -0,0 +1,40 @@
+import { Entity, Column, PrimaryGeneratedColumn, CreateDateColumn, UpdateDateColumn } from 'typeorm';
+
+@Entity()
+export class User {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column({ length: 50, unique: true })
```

Comment:
`50, ...` các giá trị hằng số thì define chung ở file constants theo từng module e nhá
còn giá trị mà tái sử dụng cho toàn project thì tạo file cùng cấp với file `app.module.ts`